src/index.js

```diff
@@ -1,6 +1,32 @@
 import './style.css';
 import addScore from './modules/addScore.js';
+import addScoreToBoard from './modules/addToLeaderboard.js';
 
 const form = document.querySelector('.form');
+// const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
 
-form.addEventListener('submit', addScore);
\ No newline at end of file
+// get game id
+// const getGameId = () => {
+//     fetch(baseUrl+"games/", {
+//       method: "POST",
+//       body: JSON.stringify({
+//         name: "Wilson's game",
+//       }),
+//       headers: {
+//         "Content-type": "application/json; charset=UTF-8",
+//       },
+//     })
+//       .then((response) => response.json())
+//       .then((json) => console.log(json));
+//   };
```

Comment:
- [ ] [OPTIONAL] Please remove the commented code to keep your repo more professional and clean.
- [ ] [OPTIONAL] Kindly make sure that the `score` input can only accept numeric values as now I can add a score as a regular text.